recipe/meta.yaml

```
@@ -68,25 +78,43 @@ outputs:
         - wheel
       run:
         - python
-        - packaging
-        - pandas ==1.5.3
-        - numpy >=1.18.5
-        - fsspec
-        - psutil
+        - packaging >=21.0
+        - pandas ==2.2.1
+        - numpy >=1.22.4
+        - fsspec >=2022.11.0
+        - psutil >=5.8.0
     test:
+      source_files:
+        - modin
       imports:
         - modin
         - modin.pandas
       requires:
-        - distributed
         - pip
+        - distributed >=2.22.0
+        - dask >=2.22.0
+        - pytest
+        - boto3
+        - requests
+        - matplotlib-base >=3.6.1
+        - xarray >=2022.03.0
```

```suggestion
        - xarray >=2022.12.0
```
https://github.com/modin-project/modin/blob/eac21a8463d7152b3ad48a8b80d6da5ac645be6a/environment-dev.yml#L23